patches/747-mikrotik-extra-support.patch

```diff
@@ -538,6 +548,17 @@
 @@ -0,0 +1,8 @@
 +// SPDX-License-Identifier: GPL-2.0-only
 +
++#include "qcom-ipq4018-discg-5acd.dts"
++
++/ {
++        compatible = "mikrotik,discg-5acd";
++        model = "MikroTik DiscG 5 ac XL (RBDiscG-5acD)";
++};
+--- /dev/null
++++ b/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-discg-5acd.dts
+@@ -0,0 +1,8 @@
++// SPDX-License-Identifier: GPL-2.0-only
++
 +#include "qcom-ipq4018-lhgg-5acd.dts"
 +
```

Comment:
Not entirely sure what's going on here, but looks like the patches are not creating the DTS files correctly. Please check the actual files once the patch is applied.

The qcom-ipq4018-lhgg-5acd-xl.dts now seems to refer to discg and lhgg-xl refers to discg. The xl one shouldnt be changed at all, and the discg one should be including similar metadata but with the different name.